instance/models/load_balancer.py

```diff
@@ -103,20 +111,16 @@ class LoadBalancingServer(ValidateModelMixin, TimeStampedModel):
     accepts_new_backends = models.BooleanField(default=True)
     # A random postfix appended to the haproxy configuration file names to make collisions
     # impossible.
-    fragment_name_postfix = models.CharField(max_length=8, blank=True)
+    fragment_name_postfix = models.CharField(
+        max_length=8,
+        blank=True,
+        default=functools.partial(generate_fragment_name, length=8),
```

Comment:
@bradenmacdonald The 8 passed in as `length` parameter should match the `max_length` above.  I intentionally did it this way so that the repetition is at least contained in three lines of code.